components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/java/org/wso2/carbon/identity/sso/saml/ui/SAMLSSOUIConstants.java

```diff
@@ -25,6 +25,24 @@
     public static final String CONFIG_CLIENT = "SAMLSSOConfigServiceClient";
     public static final String STARTING_SERVICE_PROVIDERS_DATA_PAGE = "starting_page";
     public static final String SERVICE_PROVIDERS_DATA_PAGE_COUNT = "page_count";
+    public static final String ENABLE_ATTRIBUTE_PROFILE = "enableAttributeProfile";
+    public static final String ENABLE_SINGLE_LOGOUT = "enableSingleLogout";
+    public static final String SLO_RESPONSE_URL = "sloResponseURL";
+    public static final String SLO_REQUEST_URL = "sloRequestURL";
+    public static final String ENABLE_RESPONSE_SIGNATURE= "enableResponseSignature";
+    public static final String ENABLE_ASSERTION_SIGNATURE= "enableAssertionSignature";
+    public static final String NAME_ID_FORMAT= "nameIdFormat";
+    public static final String ENABLE_DEFAULT_ATTRIBUTE_PROFILE_HIDDEN= "enableDefaultAttributeProfileHidden";
+    public static final String ENABLE_NAME_ID_CLAIM_URI_HIDDEN= "enableNameIdClaimUriHidden";
+    public static final String NAME_ID_CLAIM= "nameIdClaim";
+    public static final String ENABLE_AUDIENCE_RESTRICTION= "enableAudienceRestriction";
```

Comment:
Fix all code formatting issues below including what's there in this particular line.